methods/concat/tests.js

```diff
@@ -0,0 +1,66 @@
+import test from 'ava';
+import concat from '.';
+
+test('returns the array itself', t => {
+    const ary = [1, 2, 3];
+    t.deepEqual(
+        concat(ary, [4, 5, 6]),
+        ary
+    );
+});
+
+test('append the elements in the other array', t => {
+    const ary = [1, 2, 3];
+    t.deepEqual(
+        concat(ary, [9, 10, 11]),
+        ary
+    );
+    t.deepEqual(
+        ary,
+        [1, 2, 3, 9, 10, 11]
+    );
+    t.deepEqual(
+        concat(ary, []),
+        [1, 2, 3, 9, 10, 11]
+    );
+});
+
+test('does not loop endlessly when the argument is the original array', t => {
+    const ary = ['x', 'y'];
+    t.deepEqual(
+        concat(ary, ary),
+        ['x', 'y', 'x', 'y']
+    );
+});
+
+test('appends elements to an array with that has been shifted/popped/spliced', t => {
+    const ary = [1, 2, 3, 4, 5];
+    ary.splice(0, 2);
+    ary.splice(-2, 2);
+    t.deepEqual(
+        concat(ary, [5, 6]),
+        [3, 5, 6]
+    );
+});
+
+test('concatenates the initial value when given arguments contain multiple arguments', t => {
+    const ary = [1, 2];
+    t.deepEqual(
+        concat(ary, ary, ary),
+        [1, 2, 1, 2, 1, 2, 1, 2]
+    );
+});
+
+test('returns self when given no arguments', t => {
```

Comment:
@aldnav Perhaps change the wording of the test description, since technically you gave it one argument 😅

Reply:
@arnellebalane Updated! https://github.com/arnellebalane/rbjs/pull/40/commits/caffdb62e4e46f5742595d507997e480afded424 Thank you.